src/Main.java

```diff
@@ -1,43 +1,38 @@
-// Задача1
-public static String checkYear(int year) {
-    if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
-        return year + " год является високосным.";
-    } else {
-        return year + " год не является високосным.";
+public class Main {
+    public static void main(String[] args) {
+        // Задача 1
+        Main main = new Main();
+        System.out.println(main.checkYear(2002));
+        // Задача 2
+        clientOS(1, 2019);
+        // Задача 3
+        System.out.println(main.checkDay(50));
     }
-}
-// Задача2
-public static String clientOS(int oSClient, int clientDeviceYear) {
-    if (oSClient == 0 && clientDeviceYear < 2015) {
-        return "Установите облегченную версию приложения для iOS по ссылке.";
-    } else if (oSClient == 1 && clientDeviceYear < 2015) {
-        return "Установите облегченную версию приложения для Android по ссылке.";
-    } else if (oSClient == 0 && clientDeviceYear >= 2015) {
-        return "Установите версию приложения для iOS по ссылке.";
-    } else if (oSClient == 1 && clientDeviceYear >= 2015) {
-        return "Установите версию приложения для Android по ссылке.";
+    // Задача 1
+    public String checkYear(int year) {
+        if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
```

Comment:
```suggestion
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```
скобки надо убрать, чтобы приоритет у && был выше, чем у ||